src/main/java/org/cactoos/http/HtKeepAliveResponse.java

```diff
@@ -43,44 +42,59 @@
      */
     private static final Text TEMPLATE = new JoinedText(
         "\r\n",
-        "GET %s HTTP/1.1",
-        "Host:%s",
+        "%s",
         "Connection: Keep-Alive",
         "Keep-Alive: timeout=%s, max=%s"
     );
 
     /**
      * Ctor.
-     * @param uri Target URI
+     * @param wre The wire
      * @param mtimeout The timeout for the connection usage in milliseconds
      * @param rmax The maximum quantity of the requests within the connection
      *  timeout
-     * @todo #20:30min Replace the HtWire by HtKeepAliveWire which should
-     *  support the <em>Keep-Alive</em> header. It should has at least 1 `ctor`
-     *  with 3 parameters:
-     *  - `wire` which represents the HTTP connection;
-     *  - `timeout` in milliseconds;
-     *  - `rmax` the max quantity of within the timeout.
-     *  In case the quantity of requests is greater than rmax but takes less
-     *  time than `timeout`, the connection should be reset by the server.
-     *  Remove the suppresing of PMD errors due to unused private fields.
+     * @param req The request
+     * @checkstyle ParameterNumberCheck (58 lines)
      */
     public HtKeepAliveResponse(
-        final URI uri, final long mtimeout, final int rmax
+        final Wire wre, final long mtimeout, final int rmax, final String req
```

Comment:
@iakunin please, put each parameter in a new line and after that, check if the qulice parameter check problem got out.

Reply:
Looks like a hack, but it works :)

Reply:
Unfortunately, it didn't help. I found the same checkstyle-ignore annotation in [takes-framework](https://github.com/yegor256/takes-framework):

https://github.com/yegor256/takes/search?q=ParameterNumberCheck&unscoped_q=ParameterNumberCheck